### DISCUSSION TITLE
Additional Features?

### DISCUSSION THREAD

**Discussion by gitKairo** [TinsleyDevers/Zone7-Utility#1] [Q&A] [answered]
Hey there! I've been checking out your Zone7 Utility project for Minecraft, and it's super cool that this is your first real Java project!

I'm curious – if you were to add another feature to Zone7, what would it be?

Looking forward to hearing your creative thoughts!

**Comment by TinsleyDevers** [comment, marked as answer]
Thanks for writing, I'm currently working on multiple new features one of them being tracking how many mobs are killed with each tool. It would probably appear in the enchant or lore area of the tool. 

This isn't 100% confirmed as it might not fit my vision for the project.
